### PR TITLE
Hosted tracking

### PR DESCRIPTION
**Pull request by uplne**
Make sure we have proper data-link attributes + some small style fixes which got lost in merges.

@regiskuckaertz @lps88 @Calanthe

### CODE REVIEW

File: commercial/app/views/static/guardianHostedPage.scala.html [hunk 1]
```diff
@@ -108,7 +108,7 @@ <h1 class="hosted__heading hosted-tone--renault">Discover the New Renault Zoe</h
                 </div>
                 <div class="hosted__cta-wrapper">
                     <h1 class="hosted__heading hosted-tone--renault">Discover your Renault Zoe</h1>
-                    <a href="" class="advert__more button button--large button--primary hosted__cta" data-text="cta">
+                    <a href="" class="advert__more button button--large button--primary hosted__cta" data-link-name="explore-renaut-zoe-button">
```
Comment (regiskuckaertz) [inline commercial/app/views/static/guardianHostedPage.scala.html]: `renault` with an `l`

Comment (uplne) [inline commercial/app/views/static/guardianHostedPage.scala.html]: Nice catch. Copy paste

**Comment by regiskuckaertz** [comment]
🎱

**Comment by regiskuckaertz** [comment]
👍

**Pull request merged**